tests/test_with_server/test_get.py

```diff
@@ -185,6 +185,44 @@ def test_get_output_doesnt_run_delete_workunit_if_false(self, mock):
         _get_output_from_ecl_string(self.conn, script, delete_workunit=False)
         self.assertFalse(mock.called)
 
+    def test_get_output_stored_variables_change_output_same_type_string(self):
```

Comment:
this doesn;t actually test the types. it tests our parsing of the tyoes as they come back out. try having a number, and output 1 + a, then try a as a string, etc.